src/sklearndf/classification/_classification.py

```diff
@@ -117,176 +117,301 @@
 # Dummy
 #
 
-DummyClassifierDF = make_df_classifier(DummyClassifier)
+
+class DummyClassifierDF(ClassifierWrapperDF, DummyClassifier, native=DummyClassifier):
```

Comment:
I see that you introduce named argument `native` in the class declaration. As I understand this serves only a informative purpose when reading the code, right? I don't see any usage of `__init_subclass__` in parent classes.

Reply:
Check out `EstimatorWrapperDFMeta.__new__()` - the `native` argument is used to identify the native class being wrapped, and to remove the class from the MRO upon class instantiation.

Reply:
Technically, we could go just with the `native` argument and without stating the native class as a base class. This would also be cleaner as the final class ends up not subclassing the base class. The only reason we also state the native class as a base class is to enable code completion in IDEs that use static code inspection, hence do not see the fully instrumented wrapper class (which only happens at runtime)